### PR TITLE
Connect: Update Electron to 22.3.6

### PR DESCRIPTION
**Pull request by gzdunek**
Release notes https://releases.electronjs.org/release/compare/v22.3.2/v22.3.6

Backport to v12 and v11 after performing the v13 testplan.

### CODE REVIEW

**Review by ryanclark — APPROVED** [review]

**Review by ravicious — APPROVED** [review]
Could you run a tag build just to confirm it won't block the release of v13 alpha? Usually we'd test it anyway before merging, here we want to do thorough test together with the testplan. But we should at least make sure it won't block the build pipeline.

**Comment by gzdunek** [comment]
>Could you run a tag build just to confirm it won't block the release of v13 alpha? 

Done, `13.0.0-dev.gzdunek.12` builds fine, although when I try to run the app I get the following error:
`
Error: Error invoking remote method 'main-process-get-resolved-child-process-addresses': Error: Could not resolve address (unix:///Users/grzegorz/Library/Application Support/Teleport Connect/tsh.socket) for process [..]/Teleport Connect.app/Contents/MacOS/tsh.app/Contents/MacOS/tsh: the process exited with signal SIGKILL.
` 

It is a problem with tsh (maybe with code signing?), because the extracted version from the app also doesn't work.
`“tsh” is damaged and can’t be opened. You should move it to the Bin.` 
This build was done on GHA and I think not everything is working yet. 

Local build works fine.

**Comment by ravicious** [comment]
I don't know much about that, I'd ask on Slack. I say let's merge it in the meantime.

**Pull request merged**
